azure/services/managedclusters/spec.go

```diff
@@ -512,12 +513,6 @@ func (s *ManagedClusterSpec) Parameters(ctx context.Context, existing *asocontai
 		}
 	}
 
-	if managedCluster.Status.Tags != nil {
-		// tags managed separately because updating tags concurrently with agent pools' can cause the cluster
-		// to get stuck in an "Updating" state forever.
```

Comment:
This fix is being rolled out in AKS and I checked that this workaround doesn't seem necessary anymore in westcentralus. Tests may fail depending on region before it's fully rolled out.

Reply:
northeurope seems to not have been updated yet.

/retest

Reply:
@nojnhuh is this fixed now?

Reply:
Yes, in regions running v20231029.

https://releases.aks.azure.com/webpage/index.html